### ISSUE TITLE
Add Axe-Linter to CI 

### ISSUE THREAD

**Issue by M0nica** [NYPL/nypl-design-system#177]
### Description of desired behavior:

We should add Axe-Linter to the project to ensure that pull requests are linted for a11y related issues. We are already using the Axe storybook integration within NYPL Design Systems so I think the Axe-Linter would compliment it well.

We can set up the linter to check the changes within each pull request against a specific version of WCAG requirements. If any issues arise it will result in a fail status check.

Do we want the listing to happen on all files? If not, what files/types of files should be excluded? I am thinking to exclude files that are not .ts .tsx or .stories .

Related investigation: https://github.com/NYPL/nypl-design-system/issues/108

**Comment by helenvholmes** [comment]
Assigning myself temporarily to make certain that all the core contributors have the proper access to complete this ticket (whomever it gets assigned to)

**Comment by helenvholmes** [comment]
Upgraded @M0nica to admin, so whoever wants to take this on at Planning will have the correct permissions to.

**Comment by EdwinGuzman** [comment]
If there's a plan to experiment with jest rather than mocha as the test runner, you can also try out [`jest-axe`](https://github.com/nickcolley/jest-axe).

It seems like a useful addition to the [library card app](https://github.com/NYPL/nypl-library-card-app/pull/54). The Axe Linter seems great, but for the LCA, I want to find any issues before pushing to Github and explicitly show in tests that components are tested for accessibility.

**Comment by bigfishdesign13** [comment]
No longer valid.
